pt/convolutional-neural-networks.md

```diff
@@ -0,0 +1,718 @@
+
+<br>
+
+
+**9. [Face verification/recognition, One shot learning, Siamese network, Triplet loss]**
+
+&#10230; [Verificação / reconhecimento facial, Aprendizado de um tiro, Rede siamesa, Perda tripla]
+
+<br>
+
+
+**10. [Neural style transfer, Activation, Style matrix, Style/content cost function]**
+
+&#10230; [Transferência de estilo neural, Ativação, Matriz de estilo, Função de custo de estilo/conteúdo]
+
+<br>
+
+
+**11. [Computational trick architectures, Generative Adversarial Net, ResNet, Inception Network]**
+
+&#10230; [Arquiteturas de truques computacionais, Rede Adversarial Generativa, ResNet, Rede de Iniciação]
+
+<br>
+
+
+**12. Overview**
+
+&#10230; Visão geral
+
+<br>
+
+
+**13. Architecture of a traditional CNN ― Convolutional neural networks, also known as CNNs, are a specific type of neural networks that are generally composed of the following layers:**
+
+&#10230; Arquitetura de uma RNC (CNN) - Redes neurais convolucionais, também conhecidas como CNN (em inglês), são tipos específicos de redes neurais que geralmente são compostas pelas seguintes camadas:
+
+<br>
+
+
+**14. The convolution layer and the pooling layer can be fine-tuned with respect to hyperparameters that are described in the next sections.**
+
+&#10230; A camada convolucional e a camadas de pooling podem ter um ajuste fino considerando os hiperparâmetros que estão descritos na próxima seção. 
+
+<br>
+
+
+**15. Types of layer**
+
+&#10230; Tipos de camadas
+
+<br>
+
+
+**16. Convolution layer (CONV) ― The convolution layer (CONV) uses filters that perform convolution operations as it is scanning the input I with respect to its dimensions. Its hyperparameters include the filter size F and stride S. The resulting output O is called feature map or activation map.**
+
+&#10230; Camada convolucional (CONV) - A camada convolucional (CONV) usa filtros que realizam operações de convolução conforme eles escabeuan a entrada I com relação a suas dimensões. Seus hiperparâmetros incluem o tamanho do filtro F e o passo S. O resultado O é chamado de mapa de recursos (feature map) ou mapa de ativação.
+
+<br>
+
+
+**17. Remark: the convolution step can be generalized to the 1D and 3D cases as well.**
+
+&#10230; Observação: o passo de convolução também pode ser generalizado para os casos 1D e 3D.
+
+<br>
+
+
+**18. Pooling (POOL) ― The pooling layer (POOL) is a downsampling operation, typically applied after a convolution layer, which does some spatial invariance. In particular, max and average pooling are special kinds of pooling where the maximum and average value is taken, respectively.**
+
+&#10230; Pooling (POOL) - A camada de pooling (POOL) é uma operação de amostragem, tipicamente aplicada depois de uma camada convolucional, que faz alguma invariância espacial. Em particular, pooling máximo e médio são casos especiais de pooling onde o máximo e o médio valor são obtidos, respectivamente.
```

Comment:
Do you think _amostragem_ is a good translation for _downsampling_? I researched and found this term related with quality reduction, but I don't know if that makes sense in this context.

Reply:
since we don't have a better term, I think I'll let _amostragem_ (sampling) with the _downsampling_ term in parenthesis to keep the rightful term nearby